test/utils/review.ts

```diff
@@ -1,42 +1,52 @@
-import nock from 'nock'
-import { composeReviewDismissalMsg } from '../../src/msg_composer'
+import nock from "nock";
+import { composeReviewDismissalMsg } from "../../src/msg_composer";
+import { StatusCodes } from "http-status-codes";
 
-export function checkApproved(done: jest.DoneCallback, pullNumber: Number = 1): void {
-  nock('https://api.github.com')
-    .post(`/repos/tianhaoz95/approveman-test/pulls/${pullNumber}/reviews`, (body: any) => {
-      done(expect(body).toMatchObject({
-        event: 'APPROVE'
-      }))
-      return true
-    })
-    .reply(200)
-}
+export const checkApproved = (pullNumber: Number = 1): void => {
+  nock("https://api.github.com")
+    .post(
+      `/repos/tianhaoz95/approveman-test/pulls/${pullNumber}/reviews`,
+      (body: any) => {
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)